### PR TITLE
add browser name; refine preferred codec

### PR DESCRIPTION
**Pull request by cnderrauber**

### CODE REVIEW

File: src/room/utils.ts [hunk 1]
```diff
@@ -67,11 +67,33 @@ export interface ObservableMediaElement extends HTMLMediaElement {
...
+
+  return browser;
+}
+
 export function getClientInfo(): ClientInfo {
   const info = ClientInfo.fromPartial({
     sdk: ClientInfo_SDK.JS,
     protocol: protocolVersion,
     version,
+    browser: browserName(),
```
Comment (davidzhao) [inline src/room/utils.ts:96]: does it have to be set on the client side? the server should determine this from UserAgent string if possible.

Comment (cnderrauber) [reply src/room/utils.ts]: removed

**Review by davidzhao — APPROVED** [review]
Looks good. can you also add a `yarn changeset` comment describing the change? it'll make generating a changelog easier

**Comment by cnderrauber** [comment]
> Looks good. can you also add a `yarn changeset` comment describing the change? it'll make generating a changelog easier
Done

**Pull request merged**